spaceship/spec/connect_api/token_spec.rb

```diff
@@ -151,6 +151,21 @@
         expect(token.duration).to eq(200)
         expect(token.in_house).to eq(true)
       end
+
+      it "without issuer_id" do
+        expect(File).to receive(:binread).with('/path/to/file').and_return(private_key)
+        token = Spaceship::ConnectAPI::Token.create(
+          key_id: "key_id",
+          filepath: "/path/to/file",
+          duration: 200,
+          in_house: true
+        )
+
+        expect(token.key_id).to eq('key_id')
```

Comment:
Let's add a check for `issuer_id`: 
`expect(token.issuer_id).to be_nil`

Reply:
Thank you, fixed in https://github.com/fastlane/fastlane/pull/21987/commits/69c3770d8a061766869826d83e0834f868603dbf